addons.json

```diff
@@ -92,7 +94,8 @@
         "nextcloudinstitutions",
         "iqbrims",
         "dropboxbusiness",
-        "onedrivebusiness"
+        "onedrivebusiness",
+        "partial"
```

Comment:
`addons_commentable`  is array of AddonAppConfig.short_name.
It should be `rushfiles`. (if comentable)

Reply:
Fixed to `rushfiles`. Sorry for missing it.